### PR TITLE
Jquery upgrade

### PR DESCRIPTION
**Pull request by Damith88**

### CODE REVIEW

File: symfony/plugins/orangehrmAdminPlugin/modules/admin/templates/viewEducationSuccess.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
 ?>
 
 <?php 
-use_stylesheet(plugin_web_path('orangehrmAdminPlugin', 'css/viewEducationSuccess')); 
+//use_stylesheet(plugin_web_path('orangehrmAdminPlugin', 'css/viewEducationSuccess'));
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmAdminPlugin/modules/admin/templates/viewEducationSuccess.php]: better to remove commented line

File: symfony/plugins/orangehrmAdminPlugin/web/js/workShiftSuccess.js [hunk 2]
```diff
@@ -217,7 +217,7 @@ function getWorkShiftEmpInfo(url){
             // Use Jquery to get select list element
             var dropdownList = $("#workShift_assignedEmp")[0];
 
-            if ($.browser.msie) {
+            if (/Edge\/|Trident\/|MSIE /.test(window.navigator.userAgent)) {
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmAdminPlugin/web/js/workShiftSuccess.js]: may be a new global function identified is_msie()

File: symfony/plugins/orangehrmAdminPlugin/modules/admin/templates/viewEducationSuccess.php (hunk 1 above)
Comment (Damith88) [reply symfony/plugins/orangehrmAdminPlugin/modules/admin/templates/viewEducationSuccess.php]: done.

File: symfony/plugins/orangehrmAdminPlugin/web/js/workShiftSuccess.js (hunk 2 above)
Comment (Damith88) [reply symfony/plugins/orangehrmAdminPlugin/web/js/workShiftSuccess.js]: global function has been added to archive.js which is included in layout.

**Pull request merged**